headless/src/jsMain/kotlin/dev/fritz2/headless/components/combobox.kt

```diff
@@ -551,21 +558,25 @@ class Combobox<E : HTMLElement, T>(tag: Tag<E>, id: String?) : Tag<E> by tag, Op
 
     inner class ComboboxInput(tag: Tag<HTMLInputElement>) : Tag<HTMLInputElement> by tag {
 
+        private val itemActivationKeys = setOf(Keys.ArrowUp, Keys.ArrowDown, Keys.Home, Keys.End)
+
         private fun handleKeyboardSelections() {
-            val selectionKeydowns = keydowns {
-                if (shortcutOf(key) in setOf(Keys.ArrowUp, Keys.ArrowDown, Keys.Enter, Keys.Home, Keys.End))
-                    preventDefault()
+            val selectShortcuts = internalState.data.map { it.opened }.distinctUntilChanged().flatMapLatest { opened ->
+                if (opened) {
+                    keydownsIf {
+                        shortcutOf(this) in (itemActivationKeys + Keys.Enter)
+                    }.map { event ->
```

Comment:
This is not good, as the `preventDefault` call should be made inside the `keydownsIf` function in order to prevent fast enough.